src/clr-core/badge/badge.element.scss

```diff
@@ -1,73 +1,108 @@
+@import './../styles/tokens/generated/index';
 @import './../styles/mixins/utils';
+@import './../styles/mixins/mixins';
```

Comment:
You don't need the `mixins/mixins` file here. If I remember correctly, we are eventually going to get rid of that file. Currently there are duplicate mixins in those two files.`mixins/utils` should be edited to include the styles you need.

Reply:
The line height eraser is in `mixins/mixins` but not the other. I can open a separate ticket to combine them into a single file.

Reply:
Thanks. Should I leave it for now then?

Reply:
Leaving it for now b/c LHEs live there.